src/public/target/sections/Safety/Section.js

```diff
@@ -1,221 +1,198 @@
 import React from 'react';
-import { capitalize } from 'lodash';
+import _ from 'lodash';
 import Typography from '@material-ui/core/Typography';
 import Tooltip from '@material-ui/core/Tooltip';
 
 import { OtTableRF, Link, DataDownloader } from 'ot-ui';
 
+const Section = ({ symbol, data }) => {
+  const { adverseEffects, safetyRiskInfo } = data;
+  return (
+    <>
+      <Typography variant="h6">Known safety effects</Typography>
+      <DataDownloader
+        tableHeaders={effectsColumns}
+        rows={getAdverseDownloadData(adverseEffects)}
+        fileStem={`${symbol}-safety-effects`}
+      />
+      <OtTableRF columns={effectsColumns} data={adverseEffects} />
+      <Typography variant="h6">Safety risk information</Typography>
+      <DataDownloader
+        tableHeaders={riskColumns}
+        rows={getRiskDownloadData(safetyRiskInfo)}
+        fileStem={`${symbol}-risk-information`}
+      />
+      <OtTableRF columns={riskColumns} data={safetyRiskInfo} />
+    </>
+  );
+};
+
 const effectsColumns = [
   {
     id: 'organs_systems_affected',
     label: 'Main organs & systems affected',
-    renderCell: ({ organs_systems_affected: organs }) => {
-      return (
-        <ul>
-          {organs.map(organ => (
-            <li key={organ.code}>{organ.mapped_term}</li>
```

Comment:
Something might be weird here—for the first adverse effects row of the target [PTGS2](https://deploy-preview-146--platform-app.netlify.app/target/ENSG00000073756), the Elasticsearch database of release 20.04 lists
```JSON
"organs_systems_affected": [
  {
    "code": "UBERON_0002405",
    "mapped_term": "immune system",
    "term_in_paper": "Immune"
  },
  {
    "code": "UBERON_0004535",
    "mapped_term": "cardiovascular system",
    "term_in_paper": "cardiovascular system"
  }
]
```
but the GraphQL query in the Netlify deployment of this pull request returns
```JSON
"organsSystemsAffected": [
  {
    "code": "immune system",
    "mappedTerm": "Immune",
    "__typename": "SafetyCode"
  },
  {
    "code": "cardiovascular system",
    "mappedTerm": "cardiovascular system",
    "__typename": "SafetyCode"
  }
]
```

Reply:
I believe it to be a server-side issue: if I locally change the GraphQL query to request all three fields, they come back like this.
```JSON
"organsSystemsAffected": [
  {
    "code": "immune system",
    "mappedTerm": "Immune",
    "termInPaper": "UBERON_0002405",
    "__typename": "SafetyCode"
  },
  {
    "code": "cardiovascular system",
    "mappedTerm": "cardiovascular system",
    "termInPaper": "UBERON_0004535",
    "__typename": "SafetyCode"
  }
]
```

Reply:
For reference: has been addressed by [commit ece9eb1 in pull request #7 of the GraphQL API server](https://github.com/opentargets/platform-api-beta/pull/7/commits/ece9eb1ff2a561abf117d88f36aa78921491e299).